build.js

```diff
@@ -52,6 +51,8 @@ function doSelectorsBuild () {
 			warning_level: 'VERBOSE',
 			compilation_level: 'ADVANCED',
 			externs: path.resolve('./externs/IDomFacade.js'),
+			module_resolution: 'NODE',
```

Comment:
Note: we'll change this back to browser after a replace all of `s/import ... from ...';/import ... from ...**.js**';